src/main/java/graphql/annotations/processor/graphQLProcessors/GraphQLInputProcessor.java

```diff
@@ -52,27 +49,27 @@ public GraphQLInputProcessor() {
      * @return a {@link GraphQLInputType} that represents that object class
      */
 
-    public GraphQLInputType getInputType(Class<?> object, ProcessingElementsContainer container) {
+    public GraphQLInputType getInputTypeOrRef(Class<?> object, ProcessingElementsContainer container) {
         boolean considerAsInput = true;
```

Comment:
As I see it, the only purpose of this variable is to not put the prefix "input" in enums. Why don't just check it in the field building section, and give it the appropriate name there?

Reply:
Its not only for the prefix - its also for the graphQLTypeRetriever to know how to handle it (as input type or not).
However i changed it a bit and removed unnecessary lines of code